torch/csrc/distributed/rpc/message.cpp

```diff
@@ -86,6 +87,7 @@ bool Message::isResponse() const {
       MessageType::RREF_FETCH_RET == type_ || // ret on RRef::toHere()
       MessageType::EXCEPTION == type_ || // propagate back exceptions
       MessageType::RREF_ACK == type_ || // ret of other types
+      MessageType::PROPAGATE_GRADIENTS_RESP == type_ ||
```

Comment:
This is also an autograd response (for backward), right? If so, let's move it under the comment below.